codetf.json

```diff
@@ -48,8 +48,15 @@
                             }
                         ]
                     }
-                ]
-            }, 
```

Comment:
* It seems like this information is duplicative to the codemod namespace. For instance, `codeql:java/whatever` implies the some of the information here. Do we care? 
* How do we report the other stuff (the findings that were not fixed)? Should that be a subset at the root of a codemod's section of the CodeTF? E.g.:


```
"codemod" : "codeql:java/whatever",
"summary: "Description of rule/fix..",
"findings" : [
   { 
     "resultId" : "zxc-1",
     "outcome" : "FAILED_LOCATION_NOT_FOUND",
     "resultFile" : "my-codeql-sarif.sarif"
   },
   { 
     "resultId" : "zxc-2",
     "outcome" : "FAILED_SOURCE_NOT_FIXABLE",
     "resultFile" : "my-codeql-sarif.sarif"
   },
]
```

* I would record that `resultId` is a `String` and will contain vendor-specific ID values with an unpredictable character set.